src/metabase/api/common/validation.clj

```diff
@@ -0,0 +1,19 @@
+(ns metabase.api.common.validation
```

Comment:
The addition of this namespace is to resolve a circular dependency between `metabase.api.common` and `metabase.models.setting`.